configurevalidator/configurevalidator.go

```diff
@@ -0,0 +1,152 @@
+package configurevalidator
```

Comment:
`configurevalidator` is not the best name for this package. Maybe `configurator`?